panel/widgets/indicators.py

```diff
@@ -786,13 +796,8 @@ def __init__(self, **params):
 
         self.param.watch(self._update_layout, list(Viewable.param))
 
-        if self.value == 0:
-            # Hack: to give progress the initial look
```

Comment:
Definitely didn't like this hack but is this condition handled, i.e. under what configuration do you get an indeterminate bar and under what condition do you get an empty bar now?

Reply:
In the current state, Tqdm cannot have a value of None, so no indeterminate bar, nor -1, so no empty bar (explaining the trick).
I am proposing in my last commit to simply copy the params `value` and `max` from Progress and set default to 0 for value. So now Tqdm can have an indeterminate state and no hack is programmed.

Reply:
To repond to your question: now Tqdm can be indeterminate with a value of -1 (not happening in the gallery examples) and be empty with a value of 0.

Reply:
Okay, will check out the PR and play around. The Progress indicator can be used independently of Tqdm so I want to make sure that still works as expected.